istioctl/cmd/root.go

```diff
@@ -229,6 +225,24 @@ debug and diagnose their Istio mesh.
 	postInstallCmd.AddCommand(Webhook())
 	experimentalCmd.AddCommand(postInstallCmd)
 
+	analyzeCmd := Analyze()
+	hideInheritedFlags(analyzeCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(analyzeCmd)
+
+	convertIngressCmd := convertIngress()
+	hideInheritedFlags(convertIngressCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(convertIngressCmd)
+
+	dashboardCmd := dashboard()
+	hideInheritedFlags(dashboardCmd, "namespace", "istioNamespace")
+	rootCmd.AddCommand(dashboardCmd)
+
+	proxyStatusCmd := statusCommand()
+	hideInheritedFlags(proxyStatusCmd, "namespace", "istioNamespace")
```

Comment:
We need to keep `--namespace`.  It has no meaning for `istioctl proxy-status` but if the user does `istioctl proxy-status productpage-123` they can either use `-n bookinfo` or `productpage-123.bookinfo`.

Reply:
Done